### DISCUSSION TITLE
Use secret lists in .env file

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#43770] [Q&A] [answered]
Hi. I create secret with this command:
`php bin/console secrets:set DATABASE_URL`
How can I use this `DATABASE_URL` in my `.env` file?

**Comment by BahmanMehrdad** [comment, marked as answer]
`DATABASE_URL` is defined in the following path:

`config -> packages -> doctrine.yaml`

Now you can delete `DATABASE_URL` in `.env` file and it does not need to be
